plugins/woocommerce/tests/performance/requests/merchant/add-order.js

```diff
@@ -170,75 +179,429 @@ export function addOrder() {
+				[ 'post_type', 'shop_order' ],
+				[ 'referredby', '' ],
+				[ 'samplepermalinknonce', `${ sample_permalink_nonce }` ],
+				[ 'save', 'Create' ],
+				[ 'user_ID', '1' ],
+				[ 'wc_order_action', '' ],
+				[ 'woocommerce_meta_nonce', `${ woocommerce_meta_nonce }` ],
+			] );
+
+			const cotOrderParams = new URLSearchParams( [
+				[ '_ajax_nonce-add-meta', `${ ajax_nonce_add_meta }` ],
+				[ '_billing_address_1', `${ addresses_guest_billing_address_1 }` ],
+				[ '_billing_address_2', `${ addresses_guest_billing_address_2 }` ],
+				[ '_billing_city', `${ addresses_guest_billing_city }` ],
+				[ '_billing_company', `${ addresses_guest_billing_company }` ],
+				[ '_billing_country', `${ addresses_guest_billing_country }` ],
+				[ '_billing_email', `${ addresses_guest_billing_email }` ],
+				[
+					'_billing_first_name',
+					`${ addresses_guest_billing_first_name }`,
+				],
+				[ '_billing_last_name', `${ addresses_guest_billing_last_name }` ],
+				[ '_billing_phone', `${ addresses_guest_billing_phone }` ],
+				[ '_billing_postcode', `${ addresses_guest_billing_postcode }` ],
+				[ '_billing_state', `${ addresses_guest_billing_state }` ],
+				[ '_shipping_address_1', `${ addresses_guest_billing_address_1 }` ],
+				[ '_shipping_address_2', `${ addresses_guest_billing_address_2 }` ],
+				[ '_shipping_city', `${ addresses_guest_billing_city }` ],
+				[ '_shipping_company', `${ addresses_guest_billing_company }` ],
+				[ '_shipping_country', `${ addresses_guest_billing_country }` ],
+				[
+					'_shipping_first_name',
+					`${ addresses_guest_billing_first_name }`,
+				],
+				[ '_shipping_last_name', `${ addresses_guest_billing_last_name }` ],
+				[ '_shipping_phone', `${ addresses_guest_billing_phone }` ],
+				[ '_shipping_postcode', `${ addresses_guest_billing_postcode }` ],
+				[ '_shipping_state', `${ addresses_guest_billing_state }` ],
+				[ '_payment_method', `${ payment_method }` ],
+				[ '_transaction_id', '' ],
+				[ '_wp_http_referer', '' ],
+				[ '_wpnonce', `${ wpnonce }` ],
+				[ 'action', 'edit_order' ],
+				[ 'customer_user', '' ],
+				[ 'excerpt', '' ],
+				[ 'metakeyinput', '' ],
+				[ 'metavalue', '' ],
+				[ 'order_date', `${ order_date }` ],
+				[ 'order_date_hour', '01' ],
+				[ 'order_date_minute', '01' ],
+				[ 'order_date_second', '01' ],
+				[ 'order_note', '' ],
+				[ 'order_note_type', '' ],
+				[ 'order_status', 'wc-pending' ],
+				[ 'original_order_status', 'auto-draft' ],
+				[ 'post_status', 'auto-draft' ],
+				[ 'post_title', 'Order' ],
+				[ 'referredby', '' ],
+				[ 'save', 'Create' ],
+				[ 'wc_order_action', '' ],
+				[ 'woocommerce_meta_nonce', `${ woocommerce_meta_nonce }` ],
+			] );
+
+			if ( cot_status === true ) {
+				admin_update_order_base = `${ admin_update_order_base }&id=${ hpos_post_id }`;
+				admin_update_order_params = cotOrderParams.toString();
+			} else {
+				admin_update_order_params = orderParams.toString();
 			}
```

Comment:
This part undoes the changes from #37719, which are already in `trunk`. Specifically, these lines:

https://github.com/woocommerce/woocommerce/blob/246053277e3f40cdbc06ce002fcaa4d66f504e68/plugins/woocommerce/tests/performance/requests/merchant/add-order.js#L378-L384

Reply:
Nice catch @jorgeatorres . This should be addressed in https://github.com/woocommerce/woocommerce/pull/37665/commits/c0c260c42bbe1b80a4afb3b4aed91d41dd6c20e3